iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/methods/DeviceMethodTests.java

```diff
@@ -224,89 +156,6 @@ public void invokeMethodNullPayloadSucceed() throws Exception
         Assert.assertEquals(buildExceptionMessage("Unexpected status errors occurred", testInstance.deviceTestManager.client), 0, deviceTestManger.getStatusError());
     }
 
-    @Test
-    @StandardTierHubOnlyTest
-    @ContinuousIntegrationTest
-    public void invokeMethodNumberSucceed() throws Exception
```

Comment:
The only difference between this and the basic invokeMethod test is that the payload is a number. This only tests application layer method handling, which is outside the scope of SDK test responsibilities